server/sonar-web/src/main/js/apps/projectActivity/utils.js

```diff
@@ -114,7 +114,12 @@ export function generateSeries(
         if (measure.metric === 'uncovered_lines' && !isCustomGraph(graph)) {
           return generateCoveredLinesMetric(measure, measuresHistory);
         }
-        const metric = metrics.find(metric => metric.key === measure.metric);
+        let metric;
+        if (Array.isArray(metric)) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false"; some subsequent code is never executed. ([javascript:S2583](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS2583))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Foverview_to_ts&issues=AWEFHxVGEFfjxkQcQG6R&open=AWEFHxVGEFfjxkQcQG6R)

[ISSUE_KEY]: AWEFHxVGEFfjxkQcQG6R